### DISCUSSION TITLE
How to add name for harvester?

### DISCUSSION THREAD

**Discussion by Nebemires** [martomi/chiadog#285] [General]
How can i add name for harvester so telegram alert would look like "CHIA Harvesrter Daily : Hello farmer! 👋 Here's what happened in the last 1 hours:" ?

**Comment by martomi** [comment]
You can use title prefix in config to differentiate harvesters: https://github.com/martomi/chiadog/blob/main/config-example.yaml#L6

**Reply by Nebemires** [reply]
Thanks for guide and thanks for this tool!!!!
